less/about/newtab.less

```diff
@@ -2,22 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+@import "../variables.less";
 @import "../animations.less";
-@import "./common.less";
```

Comment:
Moved to https://github.com/brave/browser-laptop/pull/10881/files#diff-491dd03a03308cec37fad2ad0dc7ba67R13 on newprivatetab.js and https://github.com/brave/browser-laptop/pull/10881/files#diff-dbb992f7cd2508b46d5cd98e609be5a5R36 on newtab.js